day_2_core_cnf_clusters/updating/telco-update-ocp-update-prep.adoc

```diff
@@ -0,0 +1,58 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="telco-update-ocp-update-prep"]
+= Preparing the telco core cluster platform for update
+include::_attributes/common-attributes.adoc[]
+:context: ocp-update-prep
+
+toc::[]
+
+Typically, telco clusters run on bare-metal hardware.
```

Comment:
Optimization of content by rewriting something similar to:
`The telco clusters on bare-metal hardware require firmware updates for security, new features, and {product-title} compatibility.`